common/iso_config_test.go

```diff
@@ -65,12 +66,17 @@ func TestISOConfigPrepare_ISOChecksumURL(t *testing.T) {
 		t.Fatalf("bad: %#v, %#v", warns, err)
 	}
 
+	filePrefix := "file://"
+	if runtime.GOOS == "windows" {
+		filePrefix += "/"
```

Comment:
This seems like something we should do in a function in `iso_config.go`, or in a test helper. For example:

    i.ISOChecksumURL = filePrefix(cs_file.Name()) // Does the right thing based on OS

Also, I'm curious whether there is something for this in the std lib. There may not be since windows support lags a bit.